examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -43,24 +68,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
This notebook needs to be rerun top to bottom on a cpu.

Reply:
Results cleared and CUDA Dask Flag set to false to pass CI validation